### PR TITLE
Add redirect for cli install

### PR DESCRIPTION
**Pull request by liamawhite**
Signed-off-by: Liam White <liam@tetrate.io>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 node_modules/
-dist/
+dist/*
+!_redirects
```
Comment (taiki45) [inline .gitignore]: Can we specify with absolute path like ‘/dist/_redirects’?

Comment (liamawhite) [reply .gitignore]: unsure, let me try...

Comment (liamawhite) [reply .gitignore]: looks like it!

Comment (taiki45) [reply .gitignore]: We can add ignored files with force adding ‘-f’ option

**Comment by liamawhite** [comment]
I added your suggestion of `!/dist/_redirects`

**Pull request merged**